client/src/main/java/org/evosuite/symbolic/dse/algorithm/strategies/KeepSearchingCriteriaStrategy.java

```diff
@@ -22,12 +22,13 @@
 import org.evosuite.symbolic.dse.DSETestCase;
 
 import java.util.PriorityQueue;
+import java.util.Queue;
 
 /**
  * Interface for extra stopping conditions on the generated test suite
  *
  * @author ignacio lebrero
  */
 public interface KeepSearchingCriteriaStrategy {
-    boolean ShouldKeepSearching(PriorityQueue<DSETestCase> generatedTests);
+    boolean ShouldKeepSearching(Queue<DSETestCase> generatedTests);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Thanks for the comment, just updated it.

Reply:
Happy to help :)